.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
+  - package-ecosystem: "docker"
```

Comment:
Is this necessary? I'm not using Docker here. Unless GitHub Actions uses it behind the scenes - but wouldn't that be handled by the `package-ecosystem: "github-actions"` section?

Reply:
Ok, I added common template - will remove docker section. Fyi, if you don't have any Dockerfiles - the job would just skip it

Reply:
If it's a common template then leave it as is. No reason to deviate from the standards.